build/int.cloudbuild.yaml

```diff
@@ -63,6 +63,25 @@ steps:
       echo -n "Folders in scope for tests:"
       find . -type d -printf '%P\n'
 
+- id: resource specific fixups
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  entrypoint: bash
+  args:
+    - -c
+    - |
+      set -e
+      mkdir -p .hcledit-tmp
+      wget -P .hcledit-tmp/ https://github.com/minamijoyo/hcledit/releases/download/v0.2.17/hcledit_0.2.17_linux_amd64.tar.gz
+      checksum=$(sha256sum .hcledit-tmp/hcledit_0.2.17_linux_amd64.tar.gz | awk '{print $1}')
+      if [ "$checksum" != "5e085bd319c84c74e87b915ab2c1f95afccb2d4326be481fbe19c1d7a0eb5fee" ]; then
+        echo "Checksum verification failed!"
+        exit 1
+      fi
+      tar -xzf .hcledit-tmp/hcledit_0.2.17_linux_amd64.tar.gz -C .hcledit-tmp/
```

Comment:
There feels like there could be a better way to use this tool. Since this is a change to the int.cloudbuild.yaml, unless there's a reason you need to be in a step powered by CFT, you could use a golang-specific step to install this package from pkg.dev, rather than this binary. Or the tool could be made available in the CFT image directly.

Reply:
Hi @glasnt! Yup, this is just to POC this tool will work as needed.

I was thinking we might add to one of the CFT images, but I like your suggestion to use a golang specific step and go install. :)

Reply:
This looks good and very expedient - Thanks @glasnt!